### ISSUE TITLE
Support for referencing global class names

### ISSUE THREAD

**Issue by steadicat** [ladderlife/om-css#5]
We style most of our responsive components using global classes (`.mobile .desktop .tablet`). Currently it's not possible to reference those classes in the Garden selectors, because all class names get automatically prefixed.

We should have an option to opt out of the prefixing. I'm not sure what syntax is best. Maybe `:-global-desktop`, or `:-desktop`, or `:/desktop`, or just `:desktop`?

``` clojure
[:desktop :tablet
    [[:.root {:width          "875px"
              :flex-direction "row"}]
     [:.root-even {:flex-direction "row-reverse"
                   :text-align     "right"}]]]
```

**Comment by anmonteiro** [comment]
fixed 6470edfc5470ebfdb0c750af19b1672c20500d4a

**Comment by anmonteiro** [comment]
copied from the commit message:

introduces syntax that enables referencing "global" class names,
possibly defined in other CSS files. To refer to a global, existing
classname, use the prefix `$` instead of the normal `.`

e.g.: the following clojure snippet gets transformed into the CSS below

``` clojure
(ns my-ns.core)
(defcomponent my-component [props children]
  [:$desktop
   [:.root {:background-color "tomato"}]]
  (dom/div {:className "desktop"}
    (dom/div {:class :root}
      "Some text")))
```

``` css
.desktop .my_ns_core_my-component_root {
  background-color: tomato;
}
```
